redis/src/main/scala/zio/redis/Input.scala

```diff
@@ -126,6 +138,11 @@ object Input {
       Chunk(encodeString("LIMIT"), encodeString(data.offset.toString), encodeString(data.count.toString))
   }
 
+  final case class ListInput[-A](input: Input[A]) extends Input[List[A]] {
+    override private[redis] def encode(data: List[A]): Chunk[RespValue.BulkString] =
```

Comment:
```suggestion
    def encode(data: List[A]): Chunk[RespValue.BulkString] =
```

However, see the comment below, this might be unnecessary.